src/test/java/uk/gov/ons/census/casesvc/service/QuestionnaireLinkedServiceTest.java

```diff
@@ -342,6 +342,17 @@ public void testQidAlreadyLinkedToDifferentCaseCanBeReLinked() {
             eq(managementEvent.getEvent()),
             anyString(),
             eq(messageTimestamp));
+
+    verify(eventLogger)
+        .logCaseEvent(
+            any(Case.class),
```

Comment:
it might be tricky to do, but strictly speaking we should check that this is the right case

Reply:
theoretically this should be `testCase`